src/server/ssrServer.js

```diff
@@ -58,6 +59,7 @@ async function appPlugin(fastify) {
   fastify.register(fastifyMetrics, {
     defaultMetrics: { enabled: false },
     endpoint: null,
+    promClient: client,
```

Comment:
fastify-metrics has prom-client@14 as a dependency, so it was charing the same installed version before. With the prom-client update fastify-metrics was using its own instance of prom-client, so the metrics would not come through. This change allows us to get around that by providing our prom-client instance to fastify-metrics